.main.py

```diff
@@ -1,5 +1,5 @@
 import traceback,sys
-from src.Slack import Slack
+from src.Discord import Discord
```

Comment:
@vindennl48 Is this import needed? Doesn't look like the class is being used here at all.

Reply:
I went ahead and pulled this import out—pretty sure it's not needed.